src/cli/graph.js

```diff
@@ -126,15 +160,81 @@ export function _hackyIdentityNameReferenceDetector(
 export const graphHelp: Command = async (args, std) => {
   std.out(
     dedent`\
-      usage: sourcecred graph
+      usage: sourcecred graph [-d]
 
       Generate a graph from cached plugin data
 
       Either 'sourcecred load' must immediately precede this command or
       a cache directory must exist for all plugins specified in sourcecred.json
+
+      If the -d flag is provided, it will also compute and output
+      any resulting graph changes since the last generated graph.
       `.trimRight()
   );
   return 0;
 };
 
+function computeAndLogDiff(
+  oldWeightedGraph: WeightedGraph,
+  newWeightedGraph: WeightedGraph,
+  pluginName: string
+) {
+  const graphDiff = compareGraphs(
+    oldWeightedGraph.graph,
+    newWeightedGraph.graph
+  );
+  const horizontalRule =
+    "=============================================================";
+  if (graphDiff.uniqueNodesInFirst.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Unique Nodes in the Old graph\n${horizontalRule}`
+    );
+    console.log(graphDiff.uniqueNodesInFirst);
+  }
+  if (graphDiff.uniqueNodesInSecond.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Unique Nodes in the New graph\n${horizontalRule}`
+    );
+    console.log(graphDiff.uniqueNodesInSecond);
+  }
+  if (graphDiff.nodeTuplesWithDifferences.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Corresponding Nodes with Differences\n${horizontalRule}`
+    );
+    console.log(graphDiff.nodeTuplesWithDifferences);
+  }
+  if (graphDiff.uniqueEdgesInFirst.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Unique Edges in the Old graph\n${horizontalRule}`
+    );
+    console.log(graphDiff.uniqueEdgesInFirst);
+  }
+  if (graphDiff.uniqueEdgesInSecond.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Unique Edges in the New graph\n${horizontalRule}`
+    );
+    console.log(graphDiff.uniqueEdgesInSecond);
+  }
+  if (graphDiff.edgeTuplesWithDifferences.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Corresponding Edges with Differences\n${horizontalRule}`
+    );
+    console.log(graphDiff.edgeTuplesWithDifferences);
+  }
+  console.log(`${horizontalRule}\n  ${pluginName} - ${
+    graphDiff.graphsAreEqual ? "Unchanged" : "Summary of Changes"
+  }\n${horizontalRule}
+Unique Nodes in the Old graph: ${graphDiff.uniqueNodesInFirst.length}
+Unique Nodes in the New graph: ${graphDiff.uniqueNodesInSecond.length}
+Corresponding Nodes with Differences: ${
+    graphDiff.nodeTuplesWithDifferences.length
+  }
+Unique Edges in the Old graph: ${graphDiff.uniqueEdgesInFirst.length}
+Unique Edges in the New graph: ${graphDiff.uniqueEdgesInSecond.length}
+Corresponding Edges with Differences: ${
+    graphDiff.edgeTuplesWithDifferences.length
+  }
```

Comment:
When the graphs are equal, maybe let's skip this output since it's all 0?

Reply:
I'll change this in my second pass when adding weights.